test/NameRegistry.upgrade.t.sol

```diff
@@ -68,6 +68,21 @@ contract NameRegistryUpgradeTest is Test {
         assertEq(nameRegistryV2Proxy.number(), 42);
     }
 
+    function testUpgradeMaintainsExistingRoles(bytes32 role1, bytes32 role2, address alice, address bob) public {
+        _grant(role1, alice);
+
+        vm.assume(!nameRegistryProxy.hasRole(role2, bob));
```

Comment:
why assume here instead of just asserting?

Reply:
True - will assert